packages/unity-js-bridge/src/thirdweb-bridge.ts

```diff
@@ -335,13 +333,13 @@ class ThirdwebBridge implements TWBridge {
     const chainIdNumber = Number(chainId);
     if (walletInstance) {
       // local wallet needs to be generated or loaded before connecting
-      if (walletInstance.walletId === walletIds.localWallet) {
+      if (wallet === "localWallet") {
         await this.initializeLocalWallet(password as string);
         walletInstance.connect({ chainId: chainIdNumber });
       }
 
-      if (walletInstance.walletId === walletIds.embeddedWallet) {
-        const embeddedWallet = walletInstance as EmbeddedWallet;
+      if (wallet === "inAppWallet") {
```

Comment:
what if something still sends `embedded` ? is that possible or not possible due to how unity is set up?

Reply:
not gonna be possible after 4.11, just trying to get rid of dependency on previous walletIds thing